src/typeScript/services/auth.service.ts

```diff
@@ -26,7 +26,7 @@ class AuthService {
     try {
       const { data }: { data: IAuth[] } = await axios.get(endpointUrl);
       return data;
-    } catch (error) {
+    } catch (error: string | unknown) {
```

Comment:
exactly it is:
catch (error: Error) {
this.AppView.createToast(TOAST.ERROR(new Error(error).toString()));
}